### ISSUE TITLE
False positive when setting value on imported module

### ISSUE THREAD

**Issue by mauritsvanrees** [reinout/z3c.dependencychecker#42]
Small example:

```
from Products.Five import fiveconfigure
fiveconfigure.debug_mode = True
```

Result is:

```
Unused imports
==============
.../foo.py:1:  fiveconfigure
```

So: dependency checker wrongly complains that the import is not used.

In real life this happens here:
https://github.com/collective/collective.searchandreplace/blob/5.0/collective/searchandreplace/tests/base.py

**Comment by reinout** [comment]
Looks like a valid issue. First I've got to fix some work-related stuff, though :-)

**Comment by gforcada** [comment]
JFYI on the new implementation, as unused imports are not checked this would no longer be an issue 🙂

**Comment by reinout** [comment]
Ok. Fixed in 2.0 as you'll have to use pyflakes for import checking. We're not doing double work anymore.
